usersession/agent/rest_api.go

```diff
@@ -323,12 +324,24 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 		})
 	}
 
-	if err := c.s.notificationMgr.CloseNotification(notification.ID(finishRefresh.InstanceName)); err != nil {
+	summary := fmt.Sprintf(i18n.G("%q snap has been refreshed"), finishRefresh.InstanceName)
+	body := i18n.G("Now available to launch")
```

Comment:
This sentence sounds a bit strange but I'm not native speaker. I wonder also if it's missing "again"

Reply:
Same here, this was suggested by Oliver but I'm happy to rediscuss with him :)